src/basic/TRAP_INT-OMPTarget.cpp

```diff
@@ -45,7 +45,7 @@ Real_type trap_int_func(Real_type x,
 //
 // Define thread block size for target execution
 //
-#define NUMTEAMS 128
+#define NUMTEAMS 256
```

Comment:
We should change NUMTEAMS to 256 for all kernels. We are using blocksize = 256 for all CUDA variants. This would make comparisons more fair.